apps/src/aichat/types/shared.ts

```diff
@@ -0,0 +1,26 @@
+import {ValueOf} from '@cdo/apps/types/utils';
```

Comment:
What's the distinction between what's in "shared" and types that are imported from one type file to another (eg, `import {AiCustomizations} from './customizations';` in chatEvents.ts)?

Reply:
Haha good question 🙂 maybe "common" or "miscellaneous" is a better name - these are mostly one-off types that I couldn't figure out a good grouping for.

Reply:
Yeah maybe miscellaneous, with a comment that says "try to find somewhere else to put a new type if possible?

Reply:
The fact that there's imports from this file <-> chatEvents and the other way around (but no explicit circular import, as far as I can tell) feels a little smelly but I can't think of an alternative :)

Reply:
Yep sounds good - and FWIW, I already have a change in the works that will remove the `ChatCompletionApiResponse` type and remove that dependency. I imagine as we continue to refactor and clean things up, we can hopefully remove and/or find better places for a lot of these types.